test/inductor/test_torchinductor.py

```diff
@@ -5122,6 +5116,148 @@ def fn(x):
         )
 
 
+test_xfails = {
```

Comment:
Only dynamic shapes tests here, but this can potentially include other CPU/CUDA xfails, if needed. But I want to keep it limited to dynamic ones for now.